docs/setup_docs.sh

```diff
@@ -54,6 +65,7 @@ cd tmp
 integrate_connector_docs elasticsearch v3.0.0
 integrate_connector_docs aws v3.0.0
 integrate_connector_docs cassandra v3.0.0
+integrate_connector_docs pulsar main "docs/layouts*"
```

Comment:
```suggestion
integrate_connector_docs pulsar main "docs/layouts"
```
I know that just blindly copied the `layouts*`. So, it's actually my mistake. Sorry for that. But am I right in assuming that `docs/layouts*` was just copied from `docs/content*` without a reason? Or why do we need the asterix here? It appears that there is only a `docs/layout` subfolder in [flink-connector-pulsar:./docs/](https://github.com/apache/flink-connector-pulsar/tree/main/docs).

Reply:
> But am I right in assuming that docs/layouts* was just copied from docs/content* without a reason? 

Correct :) Fixed